include/functions.php

```diff
@@ -73,3 +113,25 @@ function findStartDateOfNextPeriod ($lastStartDate, $cycleLengthInDays) {
     $nextPeriodStartDate = date_add($lastStartDate, $cycleLengthInDays);
     return $nextPeriodStartDate;
 }
+
+
+function getDateToStoreInSession () {
+    if (!isset($_SESSION['date']) ||  isset($_POST['now'])){
+        $monthDisplayed = new DateTime();
+        return $monthDisplayed;
+    }
+    
+    else if (isset ($_POST['prev'])) {
+        $monthInterval = new DateInterval("P1M");
```

Comment:
Some duplicate code here --> the month interval is needed for both cases: adding or subtracting a month for 'previous' or 'next' month cases